### DISCUSSION TITLE
Delight: How do we help users on web add contacts?

### DISCUSSION THREAD

**Discussion by daniellekirkwood** [element-hq/element-meta#848] [Ideas]
Users new to Element do not have "friends" or "contacts" to message yet. They must either know the MXID of the person/room they'd like to communicate with, or join a room through the explore feature.

We know from our research that new users find an "empty" Element little cause to return - the easier we can make it for them to populate the app with connections, rooms, and content the better. 

How might we provide an "Add Contacts" feature to onboarding that connects new users to people they already know? 
(On mobile we ask users to grant permissions to contacts book)

**Comment by t3chguy** [comment]
What are contacts in Matrix?

Related vector-im/element-meta#341

**Comment by daniellekirkwood** [comment]
> What are contacts in Matrix?

Great question - maybe "Contacts" don't exist in Matrix but for our users we need to provide a way to 'find their friends'. On Mobile users are able to grant access to their contacts book and that enables them to see others that they may want to message. I'm updated this issue now.

**Comment by germain-gg** [comment]
There is a contact picker API that exists, https://developer.mozilla.org/en-US/docs/Web/API/Contact_Picker_API but is unfortunately only available for mobile users
From where I stand, if we want that feature we need to build connectors to people's contact list, which is quite a big undertaking

**Comment by t3chguy** [comment]
> They must either know the MXID of the person

On the web an e-mail can also be used, on mobile I believe phone numbers can, though matrix.org stopped accepting phone numbers being added to accounts so that may impact that severely
